thirdParty/cmake-modules/FindmallocMC.cmake

```diff
@@ -13,7 +13,7 @@
 # To provide a hint to this module where to find the mallocMC installation,
 # set the MALLOCMC_ROOT environment variable. You can also set the
 # MALLOCMC_ROOT CMake variable, which will take precedence over the environment
-# variable.
+# variable. Boths hints are preferred over the DEFAULT_PATHS.
```

Comment:
oh no, that's in the subtree. I won't fix it in this pull but will add a PR upstream.